pkg/pcrbruteforcer/reproduce_event_log.go

```diff
@@ -20,12 +20,14 @@ import (
 // SettingsReproduceEventLog defines settings for internal bruteforce algorithms used in ReproduceEventLog
 type SettingsReproduceEventLog struct {
 	SettingsBruteforceACMPolicyStatus
+	DisabledEventsMaxDistance uint64
 }
 
 // DefaultSettingsReproduceEventLog returns recommended default PCR0 settings
 func DefaultSettingsReproduceEventLog() SettingsReproduceEventLog {
 	return SettingsReproduceEventLog{
 		SettingsBruteforceACMPolicyStatus: DefaultSettingsBruteforceACMPolicyStatus(),
+		DisabledEventsMaxDistance:         2, // arbitrary value based on previous experience, hoping to handle within a second; TODO: add benchmarks
```

Comment:
We drop from 5 to 2 without a comment about the fact we're doing so in the commit message.

Reply:
Yeah. AMD has so many measurements that bruteforcing becomes too expensive computationally. Because of that I had to drop the depth of bruteforcing.